test/helpers/kubectl.go

```diff
@@ -578,6 +579,72 @@ func (kub *Kubectl) WaitCleanAllTerminatingPods() error {
 	return err
 }
 
+// ApplyNetworkPolicyUsingAPI apply a Kubernetes network policy using the API
+// and waits up  until timeout seconds for the policy to be applied in all
+// Cilium endpoints. Returns an error if the policy is not imported before the
+// timeout is exceeded.
+func (kub *Kubectl) ApplyNetworkPolicyUsingAPI(namespace string, networkPolicy *networkingv1.NetworkPolicy) error {
+	revisions := map[string]int{}
+
+	kub.logger.Infof("Creating a new policy %q", networkPolicy)
+	pods, err := kub.GetCiliumPods(KubeSystemNamespace)
+	if err != nil {
+		return err
+	}
+
+	for _, v := range pods {
+		revi, err := kub.CiliumPolicyRevision(v)
+		if err != nil {
+			return err
+		}
+		revisions[v] = revi
+		kub.logger.Infof("CiliumPolicyAction: pod '%s' has revision '%v'", v, revi)
```

Comment:
Remove `CiliumPolicyAction` from this and all log messages in this function. Given that we use the `<function name>: log message` convention in our code, and this function is not named `CiliumPolicyAction`, you could also replace it with `ApplyNetworkPolicyUsingAPI`.